src/client/pages/MovieDetailsPage/MovieDetailsPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {PureComponent} from 'react';
+import {connect} from 'react-redux';
+
+import {MovieDetails} from '../../components/MovieDetails';
+import {getMovie} from '../../actions/movie';
+
+class MovieDetailsPage extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            hasError: false
+        };
+    }
+
+    componentDidMount() {
+        const {match} = this.props;
+
+        this.props.getMovie(match.params.id);
```

Comment:
i would suggest to use lodash `get` function for such case of `id` getting. In such case we will avoid "cannot read property id of undefined, etc."

```
import { get } from 'lodash';

...

const movieId = get(match, 'params.id', 0);
```